trite-osprey/backend/db.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+
+if(fs.existsSync('.env') ){
+  require('dotenv').config()
+}
+
+const connectingString = process.env.DATABASE_URL
```

Comment:
i would take out the empty lines between your variable declarations at the top of your file